module/zstd/Makefile.in

```diff
@@ -17,6 +17,12 @@ ccflags-y += $(ZFS_MODULE_CFLAGS) $(ZFS_MODULE_CPPFLAGS)
 # Suppress unused but set variable warnings often due to ASSERTs
 ccflags-y += $(NO_UNUSED_BUT_SET_VARIABLE)
 
+# Quiet warnings about frame size due to unused code in unmodified zstd lib
+ccflags-y += -Wframe-larger-than=18000
```

Comment:
This needs to be 20480, it's still failing in debug build

Reply:
meeeh. revert fail. thank you!